src/api/app/views/webui/status_messages/new.html.haml

```diff
@@ -0,0 +1,24 @@
+- @pagetitle = 'New Status Message'
+.row
+  .col
+    .card
+      .card-body
+        %h3= @pagetitle
+        = form_for(StatusMessage.new, url: status_messages_path, method: 'post') do |f|
+          .form-group
+            = f.label(:message) do
+              Message:
+              %abbr.text-danger{ title: 'required' } *
+            = f.text_area(:message, rows: 3, required: true, class: 'form-control')
+          .form-group
+            = f.label(:severity) do
+              Severity:
+              %abbr.text-danger{ title: 'required' } *
+            %p With "announcement" severity the message will be displayed on the top and all over the web site and requires acknowledgement.
+            - options = StatusMessage.severities.keys.map { |severity| [severity.humanize, severity] }
+            = f.select(:severity, options_for_select(options), {}, class: 'custom-select')
+          .form-group
+            = f.label(:communication_scope, 'Communication Scope:')
+            - options = StatusMessage.communication_scopes.keys.map { |scope| [scope.humanize, scope] }
+            = f.select(:communication_scope, options_for_select(options), {}, class: 'custom-select')
+          = f.submit 'Accept', class: 'btn btn-primary'
```

Comment:
The action of this form is to create, so we should name the button `Create`

Reply:
done!